src/components/PreferencesBodyContainerRuntime.vue

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+import Vue from 'vue';
+import { ContainerEngine } from '@/config/settings';
+import EngineSelector from '@/components/EngineSelector.vue';
+import InputLegend from '@/components/form/InputLegend.vue';
+
+export default Vue.extend({
+  name:       'preferences-body-container-runtime',
+  components: { EngineSelector, InputLegend },
+  data() {
+    return { containerEngine: ContainerEngine.CONTAINERD };
+  },
+  methods: {
+    onChangeEngine(desiredEngine: ContainerEngine) {
+      this.containerEngine = desiredEngine;
+      this.$emit('container-runtime-change', desiredEngine);
+    },
+  }
+});
+</script>
+
+<template>
+  <div class="preference-body">
+    <input-legend>
+      <template #legend>
+        {{ t('containerRuntime.label') }}
+      </template>
+      <engine-selector
```

Comment:
We're working with unique constraints of a limited window height and no scrolling. Once approach to deal with this was to de-emphasize labels used for preferences by reducing their size. 

I think this ends up looking so jarring because of the inconsistent styling applied to labels and checkboxes, for example:

![image](https://user-images.githubusercontent.com/835961/175753193-a8db0d51-db47-4bee-a82d-a06c849c8bb7.png)

VS

![image](https://user-images.githubusercontent.com/835961/175753205-57a3eb13-57e7-44aa-aac8-005f68010cfb.png)

Relying too much on font size to denote hierarchy will lead to primary content that's too large for our use case. Similar to checkboxes above, I think we can emphasize the radio group label by de-emphasizing each radio button label.